marko.json

```diff
@@ -1,6 +1,7 @@
 {
   "taglib-id": "ebayui-core",
   "tags-dir": "./src/components",
+  "taglib-imports": ["./src/components/ebay-icon/marko.json"],
```

Comment:
Looks like `ebay-icon` is still exposed publically. We should move that into `./src/components/components` (the internal components).

Reply:
I still want to expose it publicly because of migrators. Since all the new attributes are internal it should be fine being public. I also need a place to show the examples for it.
Once we get rid of the migrators I will move the code to that components subdirectory.